arc.tex

```diff
@@ -0,0 +1,110 @@
+In addition, Rubin Observatory staff will use the Development/Integration enclave to maintain the Observatory's software tools and systems and to develop new versions of them.
+
+These enclaves are further described here and for each a series of subsections explore :
+
+
+\begin{enumerate}
+\item Threats and Security infrastructure
+\item Disaster recovery
+\end{enumerate}
+
+\subsection{Prompt \gls{Enclave}}
+
+The Prompt enclave receives images from the Observatory facilities in Chile via a Long Haul Network connection.
+It stores these and processes them into Prompt data products of three main types:
+alerts for things that have moved or changed, measurement catalogs, and processed images.
+The alerts can be further subdivided into \emph{streak} alerts for objects that have moved a long distance and \emph{non-streak} alerts for all other objects.
+Measurements in the catalogs follow the same subdivision. Images may be \gls{Commissioning} images used for testing and characterizing the Observatory systems, normal science images without significant \emph{streaks}, or delayed science images that do contain significant \emph{streaks}.
+
+\emph{Streak} alerts are transmitted to an \gls{Alert} Vetting System (AVS) located at a Trusted Broker Facility at  Lawrence Livermore National Lab.
+\emph{Non-streak} alerts and \emph{streak} alerts approved by the \gls{AVS} are to be published to the world at large within 60 seconds of the original raw image being taken.
+Normal science images are made available to data rights holders in the \gls{DAC} after an 80 hour embargo period.
+Delayed science images, as identified by the \gls{AVS}, and any unapproved \emph{streak} measurements taken from them are made available after a 10 day embargo period. Commissioning images are made available to data rights holders after a 30 day embargo period.
+
+All Prompt data products are checked for quality by automated systems but also by human operators from the Rubin Observatory staff, who have access to all images and data products in order to perform this function.
+
+\subsubsection{ Threats and Security infrastructure}
+The obvious threat surfaces here are :
+\begin{enumerate}
+\item Transmission of Data from Chile. IPSec built into the routers will be used on the \gls{LHN}. \citeds{DMTN-108} discusses threats in this realm a little more.
+\item Transmission to \gls{LLNL}.  This will be over internet using TLS.
+\item Staff access for \gls{QA}. All the usual user threats such as phishing apply - these users are however governed bu SLAC security policies. \action{RD}{March 2022}{ Need ref to SLAC security polices for users .. i.e. FACTS and all that .. }
+\item QA tools. The web accessible QA tools should have a threat analysis although they will be behind SLAC \gls{VPN} and 2FA.
```

Comment:
Seems a bit passive. A threat analysis would be conducted by who? Also I think QA tools is covered by staff access really, as they are all internal and the people are using the tools.

Reply:
ok epxlicit and noted its staff